ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -1912,6 +1912,24 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-ui
     workflow: cucushift-installer-rehearse-gcp-upi
+- as: ibmcloud-ipi-byo-kms-f28
+  cron: 53 6 26 * *
+  steps:
+    cluster_profile: ibmcloud-qe
+    env:
+      E2E_RUN_TAGS: '@amd64 and @ibmcloud-ipi and @network-ovnkubernetes and not @fips'
```

Comment:
The `network-ovnkubernetes` tag is being removed in https://github.com/openshift/release/pull/47019, so let's remove it from here

Reply:
And also for the other new added jobs.